substrafl/dependency.py

```diff
@@ -12,6 +12,10 @@
 class Dependency(BaseModel):
     """Dependency pydantic class.
 
+
+    .. note:: If you are using your current package as a local dependencies, be aware that folders named
+        **local-worker** or with **tmp_substrafl** as prefix are ignored during the installation.
```

Comment:
Is it possible to template this string with `substra.sdk.backends.local.LOCAL_WORKER_DIR` and `TMP_SUBSTRAFL_PREFIX`? 🤔